README.md

```diff
@@ -44,49 +44,59 @@ conda env create -f environment.yml
 conda activate simulate_expression_compendia
 ```
 
-R scripts were run using R 3.6.3 
+Also need to pip install modules from this repository:
+```
+pip install .
+```
+
+Note: R scripts were run using R 3.6.3 
 
 ## How to run this simulation using your own data
 
 In order to run this simulation on your own gene expression data the following steps should be performed:
 
-First we need to set up your local repository: 
+First you need to set up your local repository: 
 1. Clone the ```simulate-expression-compendia``` repository
 2. Set up conda environment using the command above
 3. Create a new analysis folder in the main directory. This is equivalent to the ```Pseudomonas``` directory
-4. Within your analysis folder create ```data/``` directory and ```input/```, ```metadata/``` subdirectories
-5. Copy ```Pseudomonas_sample_lvl_sim.ipynb``` and ```Pseudomonas_experiment_lvl_sim.ipynb``` into your respective folder.  
+4. Copy ```Pseudomonas_sample_lvl_sim.ipynb``` and ```Pseudomonas_experiment_lvl_sim.ipynb``` into your analysis folder. 
```

Comment:
You don't need triple backticks to highlight code, you can just use single backticks. Triple backticks are like triple quotes in python in that they let you make multiline code blocks